.github/workflows/ci_trie_benchmark.yml

```diff
@@ -0,0 +1,37 @@
+name: trie benchmark
+
+on:
+  workflow_call:
+  pull_request:
+    branches: ["**"]
+    paths:
+      - "crates/common/trie/**"
+
+permissions:
+  # deployments permission to deploy GitHub pages website
+  deployments: write
+  # contents permission to update benchmark contents in gh-pages branch
+  contents: write
+
+jobs:
+  build-ethrex-trie:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: true
+    steps:
+      - name: Populate cache
+        uses: actions/cache@v4
+        id: cache
+        with:
+          path: crates/common/trie
+          key: trie-${{ github.event.pull_request[matrix.trie-version].sha }}
```

Comment:
what is matrix.trie-version ?

Reply:
It seems to be missing the line to define the strategy matrix around line 20.

Reply:
I've removed this file since it will be handled in #2308